sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2842,17 +2842,6 @@ object SQLConf {
       .checkValue(_ >= 0, "This value must be equal to or greater than 0.")
       .createWithDefaultString("48MB")
 
-  val PYSPARK_JVM_STACKTRACE_ENABLED =
-    buildConf("spark.sql.pyspark.jvmStacktrace.enabled")
```

Comment:
This is a user facing conf right? I don't think we can just remove this outright.